.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,34 +1,11 @@
 ---
 name: Feature Request
 about: Suggest a new feature or enhancement for the APE library
-title: '[FEATURE] '
+title: ''
 labels: 'enhancement'
```

Comment:
The quotation for labels is not consistent with bug_report.md (where no quotation is used).

Reply:
Fixed, thanks!